ceph-releases/ALL/__QAT_ACCELERATION__

```diff
@@ -0,0 +1,2 @@
+echo '@ceph - memlock 204800' >> /etc/security/limits.conf && \
+    echo '@root - memlock 204800' >> /etc/security/limits.conf && \
```

Comment:
```suggestion
echo '@root - memlock 204800' >> /etc/security/limits.conf && \
```

Reply:
it is actually needed otherwise the rendered Dockerfile has some extra spaces